examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -93,7 +152,11 @@ using ConnectCallback = std::function<void(CHIP_ERROR, CastingPlayer *)>;
 class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
 {
 public:
-    CastingPlayer(CastingPlayerAttributes playerAttributes) { mAttributes = playerAttributes; }
+    CastingPlayer(CastingPlayerAttributes playerAttributes)
+    {
+        mAttributes                     = playerAttributes;
+        mCommissionerDeclarationHandler = new CommissionerDeclarationHandler();
```

Comment:
Should not need to do this initialization here. You can just initialize it where it is declared as a data member in the class.